docs/DEPLOY_ELASTIC.md

```diff
@@ -0,0 +1,46 @@
+# Abstruse CI Deploying
+
+Abstruse CI can automatically deploy to Amazon Elastic Beanstalk after a successful build.
+
+For a minimal configuration, you can add the following to your config file `.abstruse.yml`.
+
+```
+deploy:
+  - provider: elastic
+  - application: "test-application"
+  - accessKeyId: "your_aws_access_key"
```

Comment:
you provide key with the configuration? I guess everyone can see it then.

Reply:
That's just an example, read the next section, where it says that it's not recommended and that the best way is through environment variables...